microsite/sidebars.json

```diff
@@ -104,10 +104,30 @@
       }
     ],
     "Integrations": [
+      "integrations/index",
       {
         "type": "subcategory",
         "label": "GitHub",
-        "ids": ["integrations/github/org"]
+        "ids": [
+          "integrations/github/locations",
+          "integrations/github/discovery",
+          "integrations/github/org"
+        ]
+      },
+      {
+        "type": "subcategory",
+        "label": "GitLab",
+        "ids": ["integrations/gitlab/locations"]
+      },
+      {
+        "type": "subcategory",
+        "label": "BitBucket",
+        "ids": ["integrations/bitbucket/locations"]
+      },
+      {
+        "type": "subcategory",
+        "label": "Azure DevOps",
```

Comment:
should these be in alphabetical order perhaps?

Reply:
I was going for most-used-first, but maybe we don't really know that. That has the risk of appearing nonsensical/random too.